packages/aws-lambda-otel-extension/opt/otel-extension/external/helper.js

```diff
@@ -10,6 +11,37 @@ const extensionVersion = (() => {
   return require('../version');
 })();
 
+const resolveJson = (data) => {
+  try {
+    const res = JSON.parse(data);
+    return res;
+  } catch (error) {
+    return null;
+  }
+};
+const isJson = (data) => !!resolveJson(data);
+
+const stripResponseBlobData = (data) => {
+  data.isBodyExcluded = false;
```

Comment:
Is there a need to have this property set, if we do not exclude the body?

Reply:
Just so the payload going to our open search cluster is consistent.

Reply:
hm.. but we already have a problem with bloated payload. Ideally, if we send only properties that add some information, and in general backend should not expect exactly same set of properties every time.

Is this a problem for a backend to not have this property?

Reply:
We have a bloated payload problem on traces but that does not apply to req/res data. Also I would rather be explicit here than not given how small this data is.

Reply:
I will talk to max and the gang about the idea of removing this 👍

Reply:
Sure, I believe our rule of thumb should be, to keep payloads minimal and only introduce properties that have some extra information to describe.

In this case `isBodyExcluded: false` doesn't add anything. Our interpretation of the data will be the same wether this property is set to `false` or is not set at all.

Reply:
This is true 🤷 I will see what the ingest gang thinks about excluding properties in this payload.